net/dnsfallback/dnsfallback.go

```diff
@@ -143,3 +177,58 @@ func getDERPMap() *tailcfg.DERPMap {
 
 //go:embed dns-fallback-servers.json
 var staticDERPMapJSON []byte
+
+// cachedDERPMap is the path to a cached DERP map that we loaded from our on-disk cache
+var cachedDERPMap syncs.AtomicValue[*tailcfg.DERPMap]
```

Comment:
could just use the standard library for this as it's just a pointer... https://pkg.go.dev/sync/atomic#Pointer